src/ast-utils/ast-value.ts

```diff
@@ -35,43 +37,43 @@ function unaryToValue(unary) {
  * converts identifier to its value
  * identifier { "type": "Identifier", "name": "Number }
  */
-function identifierToValue(identifier) {
+function identifierToValue(identifier:ESTree.Identifier):string {
   return identifier.name;
 }
 
 /**
  * Function is a block statement.
  */
-function functionDeclarationToValue(fn) {
+function functionDeclarationToValue(fn:ESTree.FunctionDeclaration) {
```

Comment:
TO me, it's unclear enough what a "value" might be that return types would help, even if it's `any`. Union types, in the case the types can be primitives or AST nodes, would be helpful too.